README.md

```diff
@@ -5,166 +5,142 @@
 [![GitHub Package](https://img.shields.io/badge/GitHub-Packages-blue?logo=github)](https://github.com/spice-labs-inc/goatrodeo/packages/)
 [![Docker Image Version (latest by date)](https://img.shields.io/docker/v/spicelabs/goatrodeo?sort=date&label=Docker%20Hub)](https://hub.docker.com/r/spicelabs/goatrodeo)
 
-**Goat Rodeo** is an open-source tool that constructs **Artifact Dependency Graphs (ADGs)** from software artifacts.  
-It powers [Spice Labs CLI](https://github.com/spice-labs-inc/spice-labs-cli) and implements the [OmniBOR](https://omnibor.io) approach to content-addressable software graphs.
+**Goat Rodeo** is an open-source tool from [Spice Labs](https://spicelabs.io) that constructs **Artifact Dependency Graphs (ADGs)** from software artifacts using [OmniBOR](https://omnibor.io) content-addressable identifiers. It can be run standalone or as part of the [Spice Labs CLI](https://github.com/spice-labs-inc/spice-labs-cli).
```

Comment:
something about 'addressable' bugs me: it's a content-hash identifier, but the hash or identifier is not what we address.